package.json

```diff
@@ -31,6 +31,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.3.0",
     "react-table": "^7.0.0-rc.15",
+    "react-toastify": "^5.5.0",
```

Comment:
I bet it is a temporary solution until we implement the notifications Joakim has designed, isn't it?

Reply:
Yeah, this is analogous to the spinning loader we added last week. Let's replace it with our own implementation when we have the time.

Reply:
Fair enough.